src/components/pages/CreateItem.tsx

```diff
@@ -0,0 +1,221 @@
+import { Button, Col, DatePicker, Form, Input, Modal, Row, Select, Upload, message } from "antd";
+import { UploadOutlined } from '@ant-design/icons';
+import { useCreateItemMutation } from "../../service/itemsApi";
+import { useGetSupplierAllDataQuery } from "../../service/inventory/inventorySupplierApi";
+const { Option } = Select;
+
+interface CreateModalProps {
+    title: string;
+    visible: boolean;
+    onCancel: () => void;
+    
+}
+
+function CreateItem ({ title, visible, onCancel } : CreateModalProps) {
+    const [form] = Form.useForm();
+    const { data } = useGetSupplierAllDataQuery();
+    const [createItem] = useCreateItemMutation();
+
+    
+    const onFinish = async(values: any) => {    
```

Comment:
Please remove any and use specific type here, you can follow this one https://github.com/bablukpik/inventory-hasura-graphql/blob/main/src/components/Products/AddProduct.tsx#L15C42-L15C42